### ISSUE TITLE
Enum Generation

### ISSUE THREAD

**Issue by joao-moonward** [kimjbstar/prisma-class-generator#52]
## Expected Behavior
It should import enums from `@prisma/client`

## Actual Behavior
It is trying to import from a file that is not generated:
`import { ProductType } from './product_type'`

## Steps to Reproduce the Problem
We can use the example in the repository:
1. `npm run generate:mysql`

## Specifications

-   Version: 0.2.8
-   Prisma Version: 4.12.0
-   Platform: Mac

## Suggestion

We could implement tests to check if the change doesn't break anything.

## After some investigation

The break seems to come from the #49 PR. 
https://github.com/kimjbstar/prisma-class-generator/blob/a6be99072d94e0347f3a5895dbfa3b196bd5df17/src/components/file.component.ts#L104

This library is broken for my use case; I had to rollback to version 0.2.7

**Comment by joao-moonward** [comment]
I reckon we should at least build the generated files for each commands:
- `generate:postgresql`
- `generate:mysql`
- `generate:mongodb`

**Comment by ifrvn** [comment]
Same here. And relation fields are also not generated currectly.

For example i have two models

```prisma
model User {
  id    Int
  posts Post[]
}

model Post {
  id    Int
  content  String
  author   User       @relation(fields: [authorId], references: [id])
  authorId Int
}
```

In generated **Post.ts** file 

![image](https://github.com/kimjbstar/prisma-class-generator/assets/21982381/4dd40992-fef4-4de3-9e8a-615957678f7a)
![image](https://github.com/kimjbstar/prisma-class-generator/assets/21982381/e4d34a22-bf90-4056-8c30-10154d3d826b)

**User.ts**

![image](https://github.com/kimjbstar/prisma-class-generator/assets/21982381/4dfc930e-dbe2-4f53-b7f2-625c2adc288d)
![image](https://github.com/kimjbstar/prisma-class-generator/assets/21982381/82e677b1-7721-4aea-9107-bdd61a33a82c)

None of those imported `./***_type` files exist

**Comment by kimjbstar** [comment]
fixed in 0.2.9
